src/stories/Meter.stories.js

```diff
@@ -0,0 +1,120 @@
+import { html } from 'lit';
+import '../components/chart';
+import argTypes from '../common/config/chartArgTypes';
+
+export default {
+  title: 'Charts/Meter',
+  component: 'kd-chart',
+  decorators: [
+    (story) => html` <div style="max-width: 600px;">${story()}</div> `,
+  ],
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/uwR7B1xbaRXA5spwPvzzFO/Florence-Release?node-id=2051%3A93&mode=dev',
+    },
+  },
+  argTypes: argTypes,
+};
+
+const args = {
+  chartTitle: 'Meter Chart',
+  description: 'Chart description.',
+  labels: ['LOW', 'MED', 'HIGH'],
+  datasets: [
+    {
+      data: [30, 30, 30],
+      borderWidth: 2,
+
+      cutout: '80%',
+      // Required needleValue to show needle
+      needleValue: 40,
+      //   cutoutPercentage: 50,
+    },
+  ],
+  options: {
+    plugins: {
+      aspectRatio: 1.7,
```

Comment:
This should be under `options`, not `plugins`, it doesn't work here. You should also move it to `/config/chartTypes/meter.js`

Reply:
done